src/components/HTMLEngineProvider/HTMLRenderers/MentionUserRenderer.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import _ from 'underscore';
+import {
+    TNodeChildrenRenderer,
+} from 'react-native-render-html';
+import Navigation from '../../../libs/Navigation/Navigation';
+import ROUTES from '../../../ROUTES';
+import Text from '../../Text';
+import Tooltip from '../../Tooltip';
+import htmlRendererPropTypes from './htmlRendererPropTypes';
+import withCurrentUserPersonalDetails from '../../withCurrentUserPersonalDetails';
+import personalDetailsPropType from '../../../pages/personalDetailsPropType';
+import * as StyleUtils from '../../../styles/StyleUtils';
+
+const propTypes = {
+    ...htmlRendererPropTypes,
+
+    /**
+     * Current user personal details
+     */
+    currentUserPersonalDetails: personalDetailsPropType.isRequired,
+};
+
+/**
+ * Navigates to user details screen based on email
+ * @param {String} email
+ * @returns {void}
+ * */
+const showUserDetails = email => Navigation.navigate(ROUTES.getDetailsRoute(email));
+
+const MentionUserRenderer = (props) => {
+    const defaultRendererProps = _.omit(props, ['TDefaultRenderer', 'style']);
+    const Renderer = props.TDefaultRenderer;
+    // We need to remove the leading @ from data as it is not part of the login
+    const loginWhithoutLeadingAt = props.tnode.data.slice(1);
+
+    const isOurMention = loginWhithoutLeadingAt === props.currentUserPersonalDetails.login;
+
+    return (
+        <Text>
```

Comment:
This happens without it:
<img width="401" alt="image" src="https://user-images.githubusercontent.com/61146594/236798425-7948187d-5d59-4dde-b682-8c3e156ec650.png">

text consumes whole line, when there is only mention inside the message

Reply:
It wasn't like that in initial PR. So I traced why it is looking like this after rebasing this branch. After bisecting for a while I found this [PR](https://github.com/Expensify/App/pull/17496) and it changes the defaultViewProps for native platforms from `styles.alignItemsStart` to `styles.dFlex` and thats why the initial `Text` component want to span whole line. I saw that you @parasharrajat were reviewer for this PR too, so maybe you know if it can be switched back for native platforms, as issue resolved by this PR was connected with pointer behaviour - so specifically desktop.

Reply:
Yeah, if it wasn't intended with that PR I think it should stay `styles.alignItemsStart`. It was changed in the first commit when web and native viewProps are the same, and than propagated into separate implementations. It's harmless (because it's invisible for user), unless you want to set the background color, as with `styles.dFlex` rendered `Text` component is as wide as whole line. 

If it was intended, we have to stick to the wrapper `Text` here.

Reply:
I'd love to try and get this merged today if we can as we have a pretty quick timeline we're working with. So if we don't get a response soon, let's go forward with what we have.

Reply:
Sounds like we decided to keep it as-is?

Reply:
I think so, I'm here in case something would have to be changed